.gitmodules

```diff
@@ -3,7 +3,7 @@
 	url = https://github.com/Azure/sonic-swss-common
 [submodule "sonic-linux-kernel"]
 	path = src/sonic-linux-kernel
-	url = https://github.com/Azure/sonic-linux-kernel
+	url = https://github.com/lguohan/sonic-linux-kernel
```

Comment:
>lguohan [](start = 26, length = 7)

Use Azure repo?

Reply:
kernel has been merged yet. Will merge it before commit this pr.